skins/index.js

```diff
@@ -352,6 +353,7 @@ export default {
   dsgn,
   easybake,
   egg,
+  "element-finance": elementfinance,
```

Comment:
You will need to add only the name here, do not add a label.
```suggestion
  elementfinance,
```